### PR TITLE
Added start apiserver and workload controller manager script

### PR DESCRIPTION
**Pull request by jshaofuturewei**
As title

### CODE REVIEW

File: hack/lib/common.sh [hunk 1]
```diff
@@ -0,0 +1,297 @@
+#!/usr/bin/env bash
```
Comment (Sindica) [inline hack/lib/common.sh]: How is this common.sh? It has a lot of code similar to arktos-up but I don't see any change in arktos-up. What is the relationship? How is the script used? If you plan to refactor arktos-up, please do it in same commit. Also please include detailed user manual.

File: hack/arktos-apiserver-partition.sh [hunk 2]
```diff
@@ -1,45 +1,45 @@
...
-}
-
-function start_apiserver {
-  echo "Running arktos-up.sh..."
-  source $(dirname "$0")/arktos-up.sh
-}
-
-"$@"
-
+#!/usr/bin/env bash
```
Comment (Sindica) [inline hack/arktos-apiserver-partition.sh]: Don't know whether there will be anything else in this script. If not, suggest rename to arktos-apiserver-up-with-partition.sh

File: hack/lib/common.sh (hunk 1 above)
Comment (jshaofuturewei) [reply hack/lib/common.sh]: I removed some common functions in arktos-up.sh
The script is for reusing the existing function in arktos-up.sh.

File: hack/arktos-apiserver-partition.sh (hunk 2 above)
Comment (jshaofuturewei) [reply hack/arktos-apiserver-partition.sh]: I suppose to run the script like
bash hack/arktos-up.sh start_apiserver begin_tenant end_tenant

Comment (Sindica) [reply hack/arktos-apiserver-partition.sh]: Please make sure the manual is checked into our repo

**Pull request merged**